### PR TITLE
Chore/push operator

### PR DESCRIPTION
**Pull request by ivanstanev**
Push the last Operator to the [community-operators fork](https://github.com/snyk/community-operators).

### CODE REVIEW

File: .circleci/config.yml [hunk 1]
```diff
@@ -537,6 +537,73 @@ jobs:
...
+          set -xeo pipefail
+
+          CURRENT_DIRECTORY=$(pwd)
+          COMMUNITY_OPERATORS_UPSTREAM_LOCATION="${CURRENT_DIRECTORY}/community-operators"
+          DEPLOY_LOCATION="${COMMUNITY_OPERATORS_UPSTREAM_LOCATION}/community-operators"
+          OPERATOR_LOCATION="${CURRENT_DIRECTORY}/snyk-operator/deploy/olm-catalog/snyk-operator"
+
+          # Configure git user and gpg key
+          echo "${OPENSHIFT_OPERATOR_SIGNING_KEY_BASE64}" | base64 -d | gpg --import
+          git config --global commit.gpgsign true
```
Comment (agranado2k) [inline .circleci/config.yml:583]: That's great! It was missing in my script! Great job! 😃

File: .circleci/config/jobs/push_operator_to_community_operators.yml [hunk 2]
```diff
@@ -0,0 +1,77 @@
...
+        cd "${COMMUNITY_OPERATORS_UPSTREAM_LOCATION}"
+        git checkout -b snyk/snyk-operator-v${NEW_OPERATOR_VERSION}
+
+        # Copy new release to branch
+        cp -r "${OPERATOR_LOCATION}/${NEW_OPERATOR_VERSION}" "${DEPLOY_LOCATION}/snyk-operator/."
+        cp "${OPERATOR_LOCATION}/snyk-operator.package.yaml" "${DEPLOY_LOCATION}/snyk-operator/."
+
+        # Create the signed commit and push
+        git add "${DEPLOY_LOCATION}/snyk-operator/*"
+        git commit -s -m "Upgrade snyk-operator to version ${NEW_OPERATOR_VERSION}"
```
Comment (shaimendel) [inline .circleci/config/jobs/push_operator_to_community_operators.yml:71]: no conventions in commit message?

File: .circleci/config/jobs/push_operator_to_community_operators.yml [hunk 3]
```diff
@@ -0,0 +1,77 @@
...
+        git push --set-upstream origin --force snyk/snyk-operator-v${NEW_OPERATOR_VERSION}
+
+  - run:
```
Comment (shaimendel) [inline .circleci/config/jobs/push_operator_to_community_operators.yml:74]: what about notifying on success? will it be too noisy?

File: .circleci/config/jobs/push_operator_to_community_operators.yml (hunk 2 above)
Comment (ivanstanev) [reply .circleci/config/jobs/push_operator_to_community_operators.yml:71]: Nope https://github.com/operator-framework/community-operators/commits/master

File: .circleci/config.yml [hunk 4]
```diff
@@ -537,6 +537,73 @@ jobs:
...
+          git checkout -b snyk/snyk-operator-v${NEW_OPERATOR_VERSION}
+
+          # Copy new release to branch
+          cp -r "${OPERATOR_LOCATION}/${NEW_OPERATOR_VERSION}" "${DEPLOY_LOCATION}/snyk-operator/."
+          cp "${OPERATOR_LOCATION}/snyk-operator.package.yaml" "${DEPLOY_LOCATION}/snyk-operator/."
+
+          # Create the signed commit and push
+          git add "${DEPLOY_LOCATION}/snyk-operator/*"
+          git commit -s -m "Upgrade snyk-operator to version ${NEW_OPERATOR_VERSION}"
+          git push --set-upstream origin --force snyk/snyk-operator-v${NEW_OPERATOR_VERSION}
```
Comment (agranado2k) [inline .circleci/config.yml:600]: I think I missed something here. But we're only pushing a new branch to the community operator repo, right? But we're not opening a Pull Request. This step is still a manual process, right?

File: .circleci/config/jobs/push_operator_to_community_operators.yml (hunk 3 above)
Comment (ivanstanev) [reply .circleci/config/jobs/push_operator_to_community_operators.yml:74]: Good point... actually may be good for the `operator upgrades`, not sure for this one!

File: .circleci/config.yml (hunk 4 above)
Comment (ivanstanev) [reply .circleci/config.yml:600]: Correct, I'll update the job description.
I'm not yet sure what's the effort to open a PR to the community-operators, may have to do this change as a separate PR

**Pull request closed without merging**